sphinx/registry.py

```diff
@@ -59,7 +59,7 @@
 class SphinxComponentRegistry:
     def __init__(self) -> None:
         #: special attrgetter for autodoc; class object -> attrgetter
-        self.autodoc_attrgettrs: dict[type, Callable[[Any, str, Any], Any]] = {}
+        self.autodoc_attrgetters: dict[type, Callable[[Any, str, Any], Any]] = {}
```

Comment:
This will break important packages such as SAGE math:

https://github.com/sagemath/sage/blob/0c390a06f725cfeef11fb7f8430a0f451250af6d/src/sage_docbuild/ext/sage_autodoc.py#L2893-L2899

I would suggest keeping `autodoc_attrgettrs` as a (writable and readable) property but emit a RemovedInSphinx90Warning whenever it is accessed.

Reply:
Oops. You're right. I'll add proxy methods.